docs/learn/smart-contract-internals/types/fully-typed-contracts.mdx

```diff
@@ -35,49 +35,44 @@ So that's what Soroban CLI gives you.
-                    time then it will be empty.
-  fetch           Fetch the hash for a given contract_name.
-                    If version is not provided, it is the most recent version.
-  fetch_metadata  Fetch metadata for a given contract_name.
-                    If version is not provided, it is the most recent version.
-  deploy          Deploys a new published contract returning the deployed contract's id.
-  help            Print this message or the help of the given subcommand(s)
+  balance         Returns the balance of `id`.
 
-Options:
-  -h, --help  Print help
-```
+                      # Arguments
 
-Like any other CLI, you can also get help for any of these subcommands using something like `smartdeploy deploy --help`. Soroban CLI again fetches the on-chain interface types, this time using it to generate a full list of all arguments to the function, and even generates examples. And just to be totally clear, you don't need to alias these commands. This would work, too:
+                      * `id` - The address for which a balance is being queried. If the
+                      address has no existing balance, returns 0.
+  ...
+  transfer        Transfer `amount` from `from` to `to`.
 
-```bash
-soroban contract invoke --id 6808e8e2cce73cfca472a8197080938b320d72e7b696e34eb52ebf17070f4ee -- deploy --help
-```
+                      # Arguments
 
-Or, if you clone [soroban-examples](https://github.com/stellar/soroban-examples) and `make build` them all, you can get CLI help for any individual contract or a specific function within a contract:
+                      * `from` - The address holding the balance of tokens which will be
+                      withdrawn from.
+                      * `to` - The address which will receive the transferred tokens.
+                      * `amount` - The amount of tokens to be transferred.
 
-```bash
-$ soroban contract invoke --id 1 --wasm target/wasm32-unknown-unknown/release/soroban_hello_world_contract.wasm -- hello --help
-Usage: hello [OPTIONS]
+                      # Events
+
+                      Emits an event with topics `["transfer", from: Address, to: Address],
+                      data = [amount: i128]`
+  ...
 
 Options:
-      --to <Symbol>
-          Example:
-            --to hello
+  -h, --help  Print help
+```
 
-  -h, --help
-          Print help (see a summary with '-h')
+Like any other CLI, you can also get help for any of these subcommands using something like `native balance --help`. Soroban CLI again fetches the on-chain interface types, this time using it to generate a full list of all arguments to the function, and even generates examples. And just to be totally clear, you don't need to alias these commands. This would work, too:
+
+```bash
+soroban contract invoke --network testnet --id CDLZFC3SYJYDZT7K67VZ75HPJVIEUVNIXF47ZG2FB2RMQQVU2HHGCYSC -- balance --help
 ```
```

Comment:
The "you don't need to alias..." part might make for a good admonition box


````suggestion
Like any other CLI, you can also get help for any of these subcommands using something like `native balance --help`. Soroban CLI again fetches the on-chain interface types, this time using it to generate a full list of all arguments to the function, and even generates examples.

:::tip

And just to be totally clear, you don't need to alias these commands. This would work, too:

```bash
soroban contract invoke --network testnet --id CDLZFC3SYJYDZT7K67VZ75HPJVIEUVNIXF47ZG2FB2RMQQVU2HHGCYSC -- balance --help
 ```

:::
````